src/wp-includes/class-wp-block-supports.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Block support flags.
+ *
+ * @package WordPress
+ * 
+ * @since 5.6.0
+ */
+
+/**
+ * Class encapsulating and implementing Block Supports.
+ *
+ * @access private
+ */
```

Comment:
Minor: This doc comment needs a `@since 5.6.0`. Best to add it to all of the class's functions as well.